src/lib/sed.c

```diff
@@ -369,6 +373,11 @@ int sed_list_lr(struct sed_device *dev, const struct sed_key *key,
 	return curr_if->list_lr_fn(dev, key, lrs);
 }
 
+int sed_issue_blocksid_cmd(struct sed_device *dev, bool hw_reset)
+{
+	return curr_if->blocksid_fn(dev, hw_reset);
```

Comment:
this needs to be check if blocksid_fn is not null -- otherwise it will crash. when blocksid_fn is null return -EOPNOTSUPP

Reply:
I will add the check.